features/old/services/multiservice.feature

```diff
@@ -31,14 +30,49 @@ Feature: Multiservice feature
   @javascript
   Scenario: Create new product: Fail scenario error message
     Given I am logged in as provider "foo.3scale.localhost"
-    And provider "foo.3scale.localhost" has "multiple_services" switch allowed
-    And service discovery is not enabled
+      And provider "foo.3scale.localhost" has "multiple_services" switch allowed
+      And service discovery is not enabled
     When I am on the provider dashboard
-    And I follow "Create Product"
-    And I fill in "Name" with "Invalid API"
-    And I fill in "System name" with "I am using spaces"
-    And I press "Create Product"
-    Then I should see the flash message "System name invalid. Only ASCII letters, numbers, dashes and underscores are allowed."
+      And I follow "Create Product"
+      And I fill in "Name" with "Invalid API"
+      And I fill in "System name" with "I am using spaces"
+      And I press "Create Product"
+    Then I should see the flash message "Product could not be created"
+
+    @javascript
+    Scenario: Create new product: with blank product name
+      Given I am logged in as provider "foo.3scale.localhost"
+        And provider "foo.3scale.localhost" has "multiple_services" switch allowed
+        And service discovery is not enabled
+      When I am on the provider dashboard
+        And I follow "Create Product"
+        And I fill in "System name" with "Less fancy API"
+        And I press "Create Product"
+      Then I should see "Name Can't be blank"
+
+    @javascript
+    Scenario: Create new product: Fail scenario error message
```

Comment:
This scenario is duplicated, assert flash and error message in 1.

Reply:
sorted